src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -452,23 +465,38 @@ private EmailWrapper generateFeedbackSessionEmailBaseForInstructors(
 
         List<EmailWrapper> emails = new ArrayList<EmailWrapper>();
 
+        /* The instructor preamble aims at informing the instructor about an email that has been sent to the students
+        of his course */
+        String instructorPreamble = generateInstructorPreamble(course.getId(), course.getName());
+
+        //The corresponding preamble for students is blank since these are the recipients of the initial email.
```

Comment:
This comment format is correct. Nothing needs to be done if the comment is to be retained.